mico-admin/src/app/app-detail/app-detail.component.html

```diff
@@ -2,6 +2,12 @@
     <div class="ma3 flex-grow-1">
         <h3>Application {{application?.name}}</h3>
         <button mat-raised-button (click)="deployApplication()">deploy</button>
+        <span *ngIf="isLatestVersion" class="mh2">
+            <button mat-raised-button (click)="promoteNextVersion()">promote next Version</button>
```

Comment:
Maybe use "create new" instead of "promote next".

Reply:
will be replaced with 'create next version'
new would not sufficient reflect the fact, this function can only be used in the latest version and is used to create the consecutive version.